packages/gatsby-remark-prismjs/src/__tests__/escape-html.js

```diff
@@ -0,0 +1,16 @@
+const escapeHtml = require(`../escape-html`)
+
+describe(`escaping html entities`, () => {
```

Comment:
One more good test would be one that tests that default escapes are used correctly when additional ones are passed.

Reply:
I'll add it!